filebeat/docs/multiline.asciidoc

```diff
@@ -106,7 +119,19 @@ Exception in thread "main" java.lang.NullPointerException
         at com.example.myproject.Bootstrap.main(Bootstrap.java:14)
 -------------------------------------------------------------------------------------
 
-To consolidate these lines into a single event in {beatname_uc}, use the following multiline configuration:
+To consolidate these lines into a single event in {beatname_uc}, use the following multiline configuration with `filestream`:
```

Comment:
Can you add "with `filestream`" to each of the other examples below for consistency?

I can't make suggestions on lines that didn't change in the diff it seems.